pkg/util/validation/override_test.go

```diff
@@ -49,6 +50,54 @@ func testLoadOverrides(filename string) (map[string]interface{}, error) {
 	return overridesAsInterface, nil
 }
 
+func TestNewOverridesManager(t *testing.T) {
+	tempFile, err := ioutil.TempFile("", "test-validation")
+	require.NoError(t, err)
+
+	defer func() {
+		// Clean up
+		require.NoError(t, tempFile.Close())
+		require.NoError(t, os.Remove(tempFile.Name()))
+	}()
+
+	_, err = tempFile.WriteString(`overrides:
+  user1:
+    limit2: 150`)
+	require.NoError(t, err)
+
+	defaultTestLimits = &TestLimits{Limit1: 100}
```

Comment:
We should really have a loud warning on this global variable that you can't expect to run multiple different tests at the same time.  Or fix the design.

Reply:
I do not see any way to get rid of Global variable from tests so added a Warning.